### PR TITLE
Closes #889: Pocket megatile error handling

### LINKED ISSUE #889: Pocket megatile error handling
We have strings for if the megatile content isn't loading. #864 is a related bug.

### PR DESCRIPTION
**Pull request by dnarcese**

### CODE REVIEW

**Comment by dnarcese** [comment]
@npark-mozilla What kind of UI tests need to be written for this? Should I write them, or does QA want to take care of it?

**Comment by npark-mozilla** [comment]
@dnarcese it would be great if you can add a simple UI tests with a new UI feature.  But looking at the title, it looks like the PR is about error handling? Would there be a UI change associated with it?

**Comment by dnarcese** [comment]
I filed issue #1111 for writing the test.

File: app/src/main/res/layout/pocket_video_mega_tile.xml [hunk 1]
```diff
@@ -10,38 +10,90 @@
...
+            android:gravity="center"
+            android:visibility="gone"
+            >
+
+            <TextView
+                android:id="@+id/pocketMegaTileLoadError"
+                android:layout_height="wrap_content"
+                android:layout_width="wrap_content"
+                android:text="@string/pocket_video_feed_failed_to_load"
+                android:layout_marginTop="16dp"
```
Comment (psymoon) [inline app/src/main/res/layout/pocket_video_mega_tile.xml]: It's generally better practice to refactor and keep layout margins in /res/values/dimens.xml

**Pull request merged**